Feature/footer

shashaehhooman opened the pull request:
Hello sir,
I finished my job in footer, would you please revise this component, and declare your next orders.
Thanks regards.
![image](https://user-images.githubusercontent.com/109917155/189817280-0a6993c3-afb1-490d-8a9f-9589f61ff0ce.png)

shashaehhooman committed:
***Tasks***
1-All footer done.

shashaehhooman committed:
***Tasks***
1-A little change for checkout

reza-madankar commented:
Please take a look at below issues:
1. Replace source code with result
2. footer need to set align, please fix it
3. why you add your image into source code?

shashaehhooman committed:
***Tasks***
1-changing text alight like orginal page.
2-change the justify.
3-deleted the extra images from the orginal folder
